lib/test/test_helper.rb

```diff
@@ -15,17 +15,17 @@ def report_bug_url(request)
   def try(property)
     false
   end
+
+  def game
```

Comment:
Had to add this to get one of the tests to pass without erroring out first. All `levels` have to be associated with a `game`, so if the goal is to create test classes that look like the real thing I'm hoping this is ok?

Reply:
I think this is fine for the reason you stated.